purchase_stock_picking_show_currency_rate/tests/test_purchase_stock_picking.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from datetime import datetime
+
+from odoo.tests import Form, common
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT as DF
+
+
+class TestPurchaseStockPicking(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.currency_usd = cls.env.ref("base.USD")
+        cls.currency_eur = cls.env.ref("base.EUR")
+        cls.account_tax = cls.env["account.tax"].create(
+            {"name": "0%", "amount_type": "fixed", "type_tax_use": "sale", "amount": 0}
+        )
+        cls.partner = cls.env["res.partner"].create({"name": "Partner test"})
+        cls.product = cls.env["product.product"].create(
+            {
+                "name": "Product Test",
+                "type": "product",
+                "list_price": 10,
+                "taxes_id": [(6, 0, [cls.account_tax.id])],
+            }
+        )
+        # Create custom rates to USD + EUR
+        today = datetime.today().strftime(DF)
```

Comment:
You can use the default Odoo tools here: https://github.com/odoo/odoo/blob/13.0/odoo/fields.py#L1694
```suggestion
        today = fields.Date.today()
```